### PR TITLE
Add attractions list page, map of Toronto image, home page takes user to attraction list page

### PR DESCRIPTION
**Pull request by lkdfount**
- created attractions list page front end
- added image for the map of Toronto
- made the search button on the home page take the user to the attractions list page

### CODE REVIEW

File: travvy/src/components/AttractionsList.vue [hunk 1]
```diff
@@ -1,14 +1,126 @@
...
+      <h1>Toronto, Canada</h1>
+      <img src="../images/toronto.jpg" alt="Picture of Toronto"/>
+      <br>
+      <h3>Description</h3>
+      <p>Toronto is the largest city in Canada, is the capital of Ontario, and is home to attractions such as the CN tower.</p>
+      <h3>COVID-19</h3>
+      <p>Active cases: 2684</p>
+      <p>Trend: Numbers increasing</p>
+      <h3>Safety Guidelines</h3>
+      <p>Toronto is in phase 2</p>
```
Comment (lucasungaretti) [inline travvy/src/components/AttractionsList.vue:17]: maybe in future sprints we can describe what "phase 2" means for people from outside

File: travvy/src/components/AttractionsList.vue [hunk 2]
```diff
@@ -1,14 +1,126 @@
...
+
+      <!-- make a list of buttons that take you to attraction pages -->
+      <!-- in later sprints this list of attractions will be pulled from the database -->
+      <h1>Personalized attractions, based on your preferences</h1>
+      <!-- let the user go back to the home page and change the info they gave -->
+      <router-link to="/Home" tag="button" class="button"><span>Change Preferences</span></router-link>
+      <br><br>
+      <div class="btn-group">
+        <router-link to="/AttractionDetails" tag="button" class="button"><span>CN Tower</span></router-link>
+        <router-link to="/AttractionDetails" tag="button" class="button"><span>Ripleys</span></router-link>
```
Comment (lucasungaretti) [inline travvy/src/components/AttractionsList.vue:30]: Aquarium lol, dont worry about this we can change later

**Comment by dyampols** [comment]
LGTM

**Comment by emilynairnn** [comment]
Reviewed this and it looks good to me, thanks Lucy! I look forward to seeing it once we connect to the database in future sprints.

**Pull request merged**